main.go

```diff
@@ -66,6 +67,19 @@ func main() {
 	s.Engine.POST("/invites/:invite_id/deny", s.invitesHandler.DenyInvite)
 	s.Engine.GET("/invites", s.invitesHandler.ListInvites)
 
+	// Notes
+	s.Engine.GET("/notes/:note_id", s.notesHandler.GetNote)
+	s.Engine.POST("/notes", s.notesHandler.CreateNote)
+	s.Engine.PATCH("/notes/:note_id", s.notesHandler.UpdateNote)
+	s.Engine.DELETE("/notes/:note_id", s.notesHandler.DeleteNote)
+	s.Engine.GET("/notes/:author_id", s.notesHandler.ListNotes)
+	s.Engine.GET("/notes/:note_id/export", s.notesHandler.ExportNote)
+
+	// Blocks
+	s.Engine.POST("/notes/:note_id/blocks", s.notesHandler.InsertBlock)
+	s.Engine.PATCH("/notes/blocks/:block_id", s.notesHandler.UpdateBlock)
```

Comment:
I think you can do `/notes/:note_id/blocks/:block_id` because I think the gin router will have trouble with that otherwise. Also, it allows to make it so blocks aren't universally unique in the future which might be handy. Also it simplifies authorisation maybe?